pandas/tests/strings/test_string_array.py

```diff
@@ -73,33 +81,38 @@ def test_string_array_numeric_integer_array(method, expected):
         ("isdigit", [False, None, True]),
         ("isalpha", [True, None, False]),
         ("isalnum", [True, None, True]),
-        ("isdigit", [False, None, True]),
```

Comment:
another dup

Reply:
Maybe can add "isnumeric" instead, which doesn't seem to be tested

Reply:
sure. "isnumeric" and other "is" functions are tested to some degree by _any_string_method fixture used in test_string_array